### ISSUE TITLE
Compiler errors when trying to build for android

### ISSUE THREAD

**Issue by Doomkyn** [aliessmael/Unity-Logs-Viewer#41]
Hi, I just imported this asset and created a reporter in my first scene.
Here's a log of compiler errors I get when I try to build for android.
Running in unity editor works fine.

![Screenshot_1](https://user-images.githubusercontent.com/9033088/71618746-52705f00-2bc1-11ea-8147-c5c8f75342df.png)

**Comment by jcs090218** [comment]
I don't think this is the error from this plugin... Try reopen the Unity or change .NET version to 4.x?
